pkgs/leak_tracker_flutter_testing/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: leak_tracker_flutter_testing
+version: 1.0.0
```

Comment:
How certain are we that the API for this package is stable enough to call it a `1.0.0`?

Reply:
It is feferenced in dev_dependencies of flutter framework. I am afraid flutter will not want to reference something that starts with 0...

Reply:
Are the apis stable or do we expect additional iteration as we explore the space? I ask because we message that the leak_tracker package itself is ~alpha level, however we've published 9 major stable versions of it. In hindsight, we should have started with an `0.1` version there and evolved from that.

If we know that the API is going to evolve before we're comfortable calling it stable than we should start with a pre-release (0.x) version. If we think that the API is stable - and looks like what we'd invite people to consume - then we could start with a 1.0.

Reply:
Another way to put it - the package version is a way to message the maturity of the package.